<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWarehouseDialog</class>
 <widget class="QDialog" name="SetWarehouseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>ИД</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputId"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Название города</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputCity"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Название улицы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputStreet"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Номер дома</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputHouse"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Почтовый индекс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputZip"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>200</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonSetWarehouse">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCloseDialog">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>SetWarehouseDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetWarehouseDialog</sender>
   <signal>finished(int)</signal>
   <receiver>SetWarehouseDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
